cmd/kube-scheduler/app/testing/testserver.go

```diff
@@ -113,20 +111,21 @@ func StartTestServer(t Logger, customFlags []string) (result TestServer, err err
 		s.CombinedInsecureServing.Metrics.Listener = listener
 		t.Logf("kube-scheduler will listen insecurely on port %d...", s.CombinedInsecureServing.BindPort)
 	}
-	config, err := s.Config()
+
+	err = s.Initialize()
```

Comment:
optional: combine this line into the if-statement.

Reply:
Done